dist/split-test.js

```diff
@@ -17,6 +17,8 @@
     }
 }(this, function() {
 
+    var DEFAULT_LIFETIME = 30 * 24 * 3600;
```

Comment:
If we're only using this in setCookie, where we're multiplying by 1000, can't we just set it here? i.e.:

 var DEFAULT_LIFETIME = 30 * 24 * 60 * 60 * 1000; // In ms